opencensus/exporters/stats/stackdriver/stackdriver_exporter.h

```diff
@@ -48,6 +48,19 @@ struct StackdriverOptions {
   // https://cloud.google.com/monitoring/api/resources
   google::api::MonitoredResource monitored_resource;
 
+  // Optional: per view Stackdriver MonitoredResource to use. Key is the view
+  // descriptor name, value is the monitored resource to use for that view.
+  //
+  // If the view name cannot be found in the map, the exporter will use the
+  // "monitored_resource" set in option, or "global" resource if that is not
+  // specified.
+  //
+  // See also:
+  // https://cloud.google.com/monitoring/api/ref_v3/rpc/google.api#google.api.MonitoredResource
+  // https://cloud.google.com/monitoring/api/resources
+  std::unordered_map<std::string, google::api::MonitoredResource>
+      per_view_monitored_resource;
```

Comment:
I would call this per_metric_monitored_resource. When views are applied on the recorded measurements the result is a metric data (I know we call it view_data but that is a small mistake). So I would simply call this per_metric_, also when we will add support for gauge and cumulative metrics this will apply to that as well.

Reply:
Done. Thanks!